tests/cypress/integration/create-term.test.js

```diff
@@ -0,0 +1,48 @@
+describe('Command: createTerm', () => {
+  beforeEach(() => {
+    cy.logout();
+    cy.login();
+    cy.deleteAllTerms();
+    cy.deleteAllTerms('post_tag');
+  });
```

Comment:
Should we delete all terms created in this test suite with `afterAll()`?

Reply:
Clearing the state before running tests is a part of [Cypress Best Practices](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks)